tools/rna_tools/intarna/intarna.xml

```diff
@@ -288,15 +304,20 @@
                         <when value="add">
                             <param name="add_output" type="select" display="checkboxes" multiple="True"
                                 label="Additional output files" help="Written in a format similar to RNAplfold unpaired probability output.">
```

Comment:
eventually, all (table) files listed here can be generated gzipped...  useful? or is plain text preferrable?!

Reply:
only would have to add ".gz" in line 179

Reply:
... and within the result retrieval regex pattern

Reply:
plain text is preferred as people can see them and use simple text manipulation tools on them ... also TSV are much better than CSV :)

Reply:
hehe.. nothing is perfect.. ;)  currently, IntaRNA produces semicolon-separated output... it's on my list to make the separator user defined, but not implemented yet.

Reply:
Can we `sed` the outputs before the collection collects the files? As part of the command block.

Reply:
huuhh.. in theory yes, but this would require a bash-for-loop since the file number and names are dynamic for the input..

Reply:
will not change the current problem from below.. :/

Reply:
gut.. beantwortet meine frage von unten.. inplace=ok